queries.sql

```diff
@@ -827,7 +827,7 @@ clists AS (
 ),
 med AS (
     DELETE FROM campaign_media WHERE campaign_id = $1
-    AND media_id IS NULL or NOT(media_id = ANY($19)) RETURNING media_id
+    AND media_id IS NULL or media_id = ANY($19) RETURNING media_id
```

Comment:
This is not correct, right? Media elements that are NOT in the given list should be deleted.

Reply:
Thank you for your explanation.  Of Course you have the right. After your tip I saw that problem here is caused by  SQL logic operator precedence. I made new pull request: https://github.com/knadh/listmonk/pull/1936